ui/client/analytics/Analytics.js

```diff
@@ -0,0 +1,12 @@
+import MatomoAnalytics from "./MatomoAnalytics";
+
+export default class Analytics {
+
+  constructor(analyticsSettings) {
+    if (analyticsSettings.engine === 'Matomo') {
```

Comment:
This should be declared as const. I mean const ENGINE_MATOMO = 'Matomo'  and next: 
```
if (analyticsSettings.engine === ENGINE_MATOMO)
```

Reply:
Done